### DISCUSSION TITLE
[package.json] Group monorepo updates with others

### DISCUSSION THREAD

**Discussion by InCogNiTo124** [renovatebot/renovate#22221] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Currently I get one PR for typescript-eslint dependency update, and another PR for updating all the other packages. I want not to recieve two pull requests for what's effectively a dependency update.

### What you tried so far.

I searched the forums and the docs for configuration, no results so far.

**Comment by PhilipAbed** [comment]
can you please share your config, repository, package.json, logs?

**Reply by InCogNiTo124** [reply]
[my config](https://github.com/InCogNiTo124/personal-website/blob/master/.github/renovate.json)

**Reply by PhilipAbed** [reply, marked as answer]
your config won't do,

try this preset https://docs.renovatebot.com/presets-group/#groupall
if you want to group all dependencies.

its ready for you to use in the extends as a preset: group:all

**Comment by PhilipAbed** [comment]
i tested [this](https://docs.renovatebot.com/presets-group/#groupall):
```
{
  "groupName": "all dependencies",
  "groupSlug": "all",
  "lockFileMaintenance": {
    "enabled": false
  },
  "packageRules": [
    {
      "groupName": "all dependencies",
      "groupSlug": "all",
      "matchPackagePatterns": [
        "*"
      ]
    }
  ],
  "separateMajorMinor": false
}
```
and it put all the monorepo with the normal dependencies in one update PR

**Reply by InCogNiTo124** [reply]
This seems like it could work, I will try it out and then I'll get back to you!
Just out of curiosity, this seems to bundle **literally all** dependencies together, a feature I'm not sure I'll always want to have. How should I change the config for renovatebot to bundle only the JS dependencies (monorepo and others) in one PR, and, say, dockerfile in another PR? Is it multiple entries in the packageRules list?

**Reply by PhilipAbed** [reply]
you can match anything, 
`matchPackagePatterns` is per package
there is https://docs.renovatebot.com/configuration-options/#matchmanagers
and https://docs.renovatebot.com/configuration-options/#matchdatasources
for that, then rename the group as you wish
